common4j/src/main/com/microsoft/identity/common/java/cache/SharedPreferencesAccountCredentialCache.java

```diff
@@ -331,6 +331,7 @@ public List<Credential> getCredentialsFilteredBy(
         final List<Credential> allCredentials = getCredentials();
 
         final List<Credential> matchingCredentials = getCredentialsFilteredByInternal(
+                allCredentials,
```

Comment:
Did we change the order of some params here? Is this a breaking change?

Reply:
We are now passing the credentials for filtering as first argument in the internal method. Not a breaking change.